### ISSUE TITLE
wrong syntax highlighting if the code contains “:”

### ISSUE THREAD

**Issue by virusdefender** [teal-language/vscode-teal#28]
version `0.5.3`

```lua
logger.infof("found jar manifest app\n%s\n", app)

local x = 1000000
local y = "xxxxx"

logger:infof("found jar manifest app\n%s\n", app)
local x = 1000000
local y = "xxxxx"
```

![image](https://user-images.githubusercontent.com/4939404/106548268-a75f7100-6549-11eb-82c4-b99de608f4a4.png)

the color of line 7 and line 8 is not correct

**Comment by pdesaulniers** [comment]
Thanks! The syntax highlighting in your example should be fixed in release 0.5.4.

**Comment by pdesaulniers** [comment]
I'm closing the issue for now, feel free to reopen it if you can still reproduce the bug.
